test/e2e/config.js

```diff
@@ -1,27 +1,24 @@
-const argv = require('yargs').argv;
+const yargs = require('yargs').argv;
 
 function browserName() {
-  if (argv) {
-    if (argv.chrome) {
+  if (process.argv) {
```

Comment:
Kunnen we de if-testen niet properder doen door yargs uitgebreider te benutten? Bijvoorbeeld: https://github.com/yargs/yargs/blob/HEAD/docs/examples.md#and-if-you-really-want-to-get-all-descriptive-about-it

Reply:
+1

Reply:
deferred